content/riak/cs/3.0.0/cookbooks/release-notes.md

```diff
@@ -132,7 +132,7 @@ supported due to changes in the manifest record).
 
 Released April 7, 2019.
 
-This is a backwards-compatible* release that updates node_package to address a recent [Product Advisory](http://docs.basho.com/riak/latest/community/product-advisories/codeinjectioninitfiles/), as well as fixes several bugs.
+This is a backwards-compatible* release that updates node_package to address a recent [Product Advisory]({{<baseurl>}}/iak/community/product-advisories/codeinjectioninitfiles/), as well as fixes several bugs.
```

Comment:
Should `{{<baseurl>}}/iak/` be `{{<baseurl>}}/riak/`?

Also, isn't it `{{<baseurl>}}riak/` and not `{{<baseurl>}}/riak/`?